diff -ur nxcompext-o/Clean.c nxcompext/Clean.c
--- nxcompext-o/Clean.c	2006-06-10 16:53:07.000000000 -0600
+++ nxcompext/Clean.c	2006-09-08 17:10:09.000000000 -0600
@@ -18,7 +18,7 @@
 #include <stdio.h>
 #include <signal.h>
 
-#include "os.h"
+#include "xorg/os.h"
 
 #include "NXlib.h"
 
diff -ur nxcompext-o/Clean.h nxcompext/Clean.h
--- nxcompext-o/Clean.h	2006-05-12 11:49:20.000000000 -0600
+++ nxcompext/Clean.h	2006-09-08 17:10:09.000000000 -0600
@@ -22,7 +22,7 @@
 extern "C" {
 #endif
 
-#include "Xlib.h"
+#include "X11/Xlib.h"
 
 int CleanXYImage(XImage *image);
 int CleanZImage(XImage *image);
diff -ur nxcompext-o/Mask.c nxcompext/Mask.c
--- nxcompext-o/Mask.c	2006-06-01 13:49:45.000000000 -0600
+++ nxcompext/Mask.c	2006-09-08 17:10:09.000000000 -0600
@@ -17,7 +17,7 @@
 
 #include <stdio.h>
 
-#include "Xlib.h"
+#include "X11/Xlib.h"
 
 #include "NXpack.h"
 
diff -ur nxcompext-o/Mask.h nxcompext/Mask.h
--- nxcompext-o/Mask.h	2006-05-12 11:49:20.000000000 -0600
+++ nxcompext/Mask.h	2006-09-08 17:10:09.000000000 -0600
@@ -22,7 +22,7 @@
 extern "C" {
 #endif
 
-#include "Xlib.h"
+#include "X11/Xlib.h"
 
 extern int MaskImage(const ColorMask *mask, XImage *src_image, XImage *dst_image);
 
diff -ur nxcompext-o/NXlib.c nxcompext/NXlib.c
--- nxcompext-o/NXlib.c	2006-09-08 17:03:52.000000000 -0600
+++ nxcompext/NXlib.c	2006-09-08 17:10:09.000000000 -0600
@@ -23,8 +23,8 @@
 
 #include "NX.h"
 
-#include "dix.h"
-#include "os.h"
+#include "xorg/dix.h"
+#include "xorg/os.h"
 
 #define NEED_REPLIES
 
@@ -35,9 +35,9 @@
 
 #define NX_TRANS_SOCKET
 
-#include "Xlib.h"
-#include "Xutil.h"
-#include "Xlibint.h"
+#include "X11/Xlib.h"
+#include "X11/Xutil.h"
+#include "X11/Xlibint.h"
 
 #include "NXlib.h"
 #include "NXproto.h"
diff -ur nxcompext-o/Pgn.c nxcompext/Pgn.c
--- nxcompext-o/Pgn.c	2006-05-17 12:38:22.000000000 -0600
+++ nxcompext/Pgn.c	2006-09-08 17:10:20.000000000 -0600
@@ -19,7 +19,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "Xutil.h"
+#include "X11/Xutil.h"
 
 #include "NXlib.h"
 
